roster2/bundle/src/bundle/roster_eu.properties

```diff
@@ -0,0 +1,115 @@
+
+# Group membership drop-down
+roster_group_ungrouped=Sin grupo
+roster_group_bygroup=Por grupo
+roster_group_unassigned=Sin asignar
+
+# Enrollment drop-down
+roster_enrollment_status_all=Todos
+
+# Participants and roles
+currently_displaying_participants=Mostrando actualmente {0} participantes
+currently_displaying_participant=Mostrando actualmente {0} participante(s)
+role_breakdown_fragment={0} con rol {1}
+
+# Filters
+enrollment_set_filter_label=Configurar
+enrollment_status_filter_label=Estado
+
+# Roster member group drop-down
+member_group_all_option=M\u00faltiple...
+
+# Print and export buttons
+print_roster=Imprimir
+print_roster_tooltip=Imprimir esta p\u00e1gina
+export_roster=Exportar
+export_roster_tooltip=Exportar esta lista
+
+# Show/hide names button
+roster_hide_names=Ocultar nombres
+roster_hide_pictures=Ocultar fotos
+
+# Official/Profile radio
+picture_source_group=Origen de la foto
+roster_show_official_pictures=Oficial
+roster_show_profile_pictures=Perfil
+
+# View types
+layout_group=Vista
+roster_card_view=Fichas
+roster_photo_view=Panel de fotos
+roster_spreadsheet_view=Lista
+
+# Profile
+profile_email=Correo
+profile_picture_alt=Foto de
+
+# Facets
+facet_picture=Foto
+facet_name=Nombre
+facet_userId=ID
+facet_pronouns=Pronombres
+facet_user_name_pronunciation=Pronunciaci\u00f3n
+facet_userProperties=Propiedades de usuario
+facet_email=Correo
+facet_role=Rol
+facet_status=Estado
+facet_credits=Cr\u00e9ditos
+facet_groups=Grupos
+facet_roster=Participantes
+
+# Page title messages
+title_msg_permissions=Definir los permisos para los roles en el sitio actual.
+no_participants=No se encontraron resultados
+
+# Permissions
+roster_permissions_role=Rol
+roster_save_button=Guardar
+roster_cancel_button=Cancelar
+permissions_header=Permisos
+
+perm-roster.viewallmembers=Ver todos los participantes
+perm-roster.viewhidden=Ver participantes ocultos
+perm-roster.export=Exportar el listado
+perm-roster.viewgroup=Ver grupos
+perm-roster.viewenrollmentstatus=Ver el estado de inscripci\u00f3n de los participantes
+perm-roster.viewprofile=Ver perfiles                  
+perm-roster.viewemail=Ver direcciones de correos
+perm-roster.viewofficialphoto=Ver fotos oficiales
+perm-roster.viewsitevisits=Ver visitas al sitio
+perm-roster.viewuserproperties=Ver propiedades de participantes
+
+groups=Grupo
+roles_label=Rol
+total_visits=Visitas totales
+last_visit=\u00daltima visita
+months=Ene,Feb,Mar,Abr,May,Jun,Jul,Ago,Sep,Oct,Nov,Dic
+no_visits_yet=Ninguno todav\u00eda
+screenreader_comboBox=Pulse Alt + cursor (arriba o abajo) para desplazarse por el men\u00fa
+name_pronunciation_not_provided=[No proporcionado]
+name_pronunciation_profile_link=Introduzca la pronunciaci\u00f3n de su nombre en el Perfil
+roster_user_groups_label=Selector donde se muestran los grupos
+
+# S2U-45
+facet_sections=Secciones
+facet_sections_category=Secciones de la categor\u00eda:\u0020
+
+# THIS FILE WILL BE TRANSLATED by: Pablo San Roman
```

Comment:
@pablosanroman I feel this file should be translated, but you can also provide the last two values.